site/gatsby-site/src/components/cite/Tools.js

```diff
@@ -162,4 +198,123 @@ function Tools({
   );
 }
 
+function RemoveDuplicateModalContents({ incident }) {
+  const [duplicateIncidentId, setDuplicateIncidentId] = useState(null);
+
+  const [deletingDuplicate, setDeletingDuplicate] = useState(false);
+
+  const [updateIncident] = useMutation(UPDATE_INCIDENT);
+
+  const [insertDuplicate] = useMutation(INSERT_DUPLICATE);
+
+  const [updateClassifications] = useMutation(UPDATE_CLASSIFICATIONS_INCIDENT_ID);
+
+  const { data: duplicateIncidentData, loading: duplicateIncidentLoading } = useQuery(
+    FIND_INCIDENT,
+    { variables: { query: { incident_id: duplicateIncidentId } } }
+  );
+
+  const duplicateIncident = duplicateIncidentData?.incident;
+
+  return (
+    <>
+      <Trans>Transfer reports to incident</Trans>{' '}
+      <input
+        type="number"
+        className="w-20 ml-1"
+        onChange={(event) => setDuplicateIncidentId(event.target.value)}
+        value={duplicateIncidentId}
+      />
+      {duplicateIncidentId && duplicateIncident ? (
+        <>
+          <p>
+            <LocalizedLink to={`/cite/${duplicateIncident.incident_id}`}>
+              Incident {duplicateIncident.incident_id} – {duplicateIncident.title}
+            </LocalizedLink>
+          </p>
+        </>
+      ) : duplicateIncidentLoading ? (
+        <div className="my-4">
+          <Spinner />
+        </div>
+      ) : (
+        <div className="my-8"></div>
+      )}
+      <Button
+        color="failure"
+        disabled={duplicateIncidentLoading || !duplicateIncident || deletingDuplicate}
+        onClick={async () => {
+          setDeletingDuplicate(true);
+          const reportIds = removeTypename(
+            duplicateIncidentData.incident.reports.concat(incident.reports)
+          ).map((report) => report.report_number);
+
+          let insertDuplicateResponse;
+
+          try {
+            insertDuplicateResponse = await insertDuplicate({
+              variables: {
+                duplicate: {
+                  duplicate_incident_number: incident.incident_id,
+                  true_incident_number: duplicateIncidentId,
+                },
+              },
+            });
+          } catch (e) {
+            console.error(insertDuplicateResponse);
+            alert(`Could not insert duplicate. Aborting.`);
+            return;
+          }
+
+          let updateIncidentResponse;
+
+          try {
+            updateIncidentResponse = await updateIncident({
+              variables: {
+                query: { incident_id: duplicateIncidentId },
+                set: { reports: { link: reportIds } },
+              },
+            });
+          } catch (e) {
+            console.error(updateIncidentResponse);
+            alert(`Could not transfer reports to incident ${duplicateIncidentId}.`);
```

Comment:
Let's use toasts instead of alerts for consistency.

Also, using toasts and passing the exception will log it to Rollbar and give us visibility on when this fails.